packages/mobile/src/home/RecentlyUsedDapps.tsx

```diff
@@ -0,0 +1,137 @@
+import { Colors } from '@celo/react-components/styles/colors'
+import fontStyles from '@celo/react-components/styles/fonts'
+import { Spacing } from '@celo/react-components/styles/styles'
+import variables from '@celo/react-components/styles/variables'
+import * as React from 'react'
+import { useState } from 'react'
+import { useTranslation } from 'react-i18next'
+import {
+  Image,
+  NativeScrollEvent,
+  ScrollView,
+  StyleSheet,
+  Text,
+  TouchableOpacity,
+  View,
+} from 'react-native'
+import { recentDappsSelector } from 'src/app/selectors'
+import { Dapp } from 'src/app/types'
+import ProgressArrow from 'src/icons/ProgressArrow'
+import { navigate } from 'src/navigator/NavigationService'
+import { Screens } from 'src/navigator/Screens'
+import useSelector from 'src/redux/useSelector'
+
+interface Props {
+  onSelectDapp(dapp: Dapp): void
+}
+
+const DAPP_ICON_SIZE = 68
+
+function RecentlyUsedDapps({ onSelectDapp }: Props) {
+  const recentlyUsedDapps = useSelector(recentDappsSelector)
+  const [currentIndex, setCurrentIndex] = useState(0)
+  const { t } = useTranslation()
+
+  const handleScroll = (event: { nativeEvent: NativeScrollEvent }) => {
```

Comment:
I'm not sure I understand the purpose of this, why are we handling the scroll like this?

Reply:
ha. there is no reason, i didn't do a good job of covering up my copy+paste job from the notification box component. this was being used to track which notification is focused but we don't need that here. i've removed!